src/Microsoft.AspNetCore.Razor.LanguageServer/Serialization/RazorConfigurationJsonConverter.cs

```diff
@@ -28,10 +28,15 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 
             var obj = JObject.Load(reader);
             var configurationName = obj[nameof(RazorConfiguration.ConfigurationName)].Value<string>();
-            var languageVersion = obj[nameof(RazorConfiguration.LanguageVersion)].Value<string>();
+            var languageVersionValue = obj[nameof(RazorConfiguration.LanguageVersion)].Value<string>();
             var extensions = obj[nameof(RazorConfiguration.Extensions)].ToObject<RazorExtension[]>(serializer);
 
-            return RazorConfiguration.Create(RazorLanguageVersion.Parse(languageVersion), configurationName, extensions);
+            if (!RazorLanguageVersion.TryParse(languageVersionValue, out var languageVersion))
+            {
+                languageVersion = RazorLanguageVersion.Version_2_1;
+            }
+
+            return RazorConfiguration.Create(languageVersion, configurationName, extensions);
```

Comment:
I'm really concerned about the fact that you took a non-shared source version of this code, and you're now maintiaining it. What's the plan to unify this?

Reply:
This is more of an artifact of being in the "alpha" phase of development. For a while we couldn't depend on Razor packages because of strong naming restrictions. Now that that is resolved I just need to go back through the code and change it all to use the proper packages. 

In this specific case I couldn't find a package for the `Microsoft.VisualStudio.LanguageServices.Blazor` so I hacked together a solution since i'll have to go back and hot swap everything else anyhow.

Reply:
Oh and we also need to change the tooling packages/projects to cross-compile for something netcoreapp2.0 compatible